app/views/account_price_group_members/search_results.html.haml

```diff
@@ -1,15 +1,15 @@
-- if params[:search_term] == '' || params[:search_term].nil?
-  %p.error= t('.error.noterm')
+- if params[:search_term] == "" || params[:search_term].nil?
```

Comment:
This could be `params[:search_term].blank?`. Or flip the if/else blocks and use `present?`